ci-operator/templates/openshift/openshift-azure/build-base-image.yaml

```diff
@@ -0,0 +1,174 @@
+# This template is being used for testing OpenShift Ansible base image builds for Azure.
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: NAMESPACE
+  required: true
+- name: CLUSTER_TYPE
+  value: "azure"
+  required: true
+- name: AZURE_VM_SIZE
+  value: "Standard_D2s_v3"
+  required: true
+- name: IMAGE_ANSIBLE
+  required: true
+- name: DEPLOY_OS
+  value: "rhel7"
+  required: true
+
+objects:
+
+# We want the cluster to be able to access these images
+- kind: RoleBinding
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ${JOB_NAME_SAFE}-image-puller
+    namespace: ${NAMESPACE}
+  roleRef:
+    name: system:image-puller
+  subjects:
+  - kind: SystemGroup
+    name: system:unauthenticated
+
+# The build pod will create an instance and install openshift
+- kind: Pod
+  apiVersion: v1
+  metadata:
+    name: ${JOB_NAME_SAFE}
+    namespace: ${NAMESPACE}
+  spec:
+    restartPolicy: Never
+    activeDeadlineSeconds: 10800
+    terminationGracePeriodSeconds: 600
+    volumes:
+    - name: shared-tmp
+      emptyDir: {}
+    - name: cluster-secrets-azure
+      secret:
+        secretName: e2e-azure-secret
+
+    containers:
+    - name: buildimage
+      image: ${IMAGE_ANSIBLE}
+      resources:
+        requests:
+          cpu: 1
+          memory: 300Mi
+        limits:
+          cpu: 3
+          memory: 4Gi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-secrets-azure
+        mountPath: /usr/share/ansible/openshift-ansible/inventory/dynamic/injected
```

Comment:
Do we need this? cant remember.

Reply:
yes we need that in order to later be able to obtain yum client certificates needed to update packages on the root VM prior to creation of the base image

```
-e @/usr/share/ansible/openshift-ansible/inventory/dynamic/injected/certs.yaml \
```